### PR TITLE
[CARE-4999] Feature - Allow iframing the theme in Prezly app

### PR DESCRIPTION
**Pull request by kudlajz**

### CODE REVIEW

File: packages/nextjs/src/next-config/index.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export function createNextConfig({ recommendedHeaders = true }: Partial<Options>
                             },
                             {
                                 key: 'Content-Security-Policy',
-                                value: 'upgrade-insecure-requests; report-uri https://csp.prezly.net/report;',
+                                value: "upgrade-insecure-requests; report-uri https://csp.prezly.net/report; frame-ancestors 'self' https://rock.prezly.com https://rock.*.prezly.net http://rock.prezly.test",
```
Comment (kudlajz) [inline packages/nextjs/src/next-config/index.ts:41]: I see the docs mention that the parent must match the protocol, but we don't have `https` locally for Prezly 🤔

Comment (camilb) [reply packages/nextjs/src/next-config/index.ts:41]: @kudlajz Is this necessary to be set here? I thought was enough for preview URLs

Comment (kudlajz) [reply packages/nextjs/src/next-config/index.ts:41]: I'm not sure what you mean.. we need to be able to iframe the newsrooms in the production app as well 🤔

Comment (camilb) [reply packages/nextjs/src/next-config/index.ts:41]: Ok, didn't know that. It is alright then.

**Review by camilb — APPROVED** [review]
👍

**Pull request merged**